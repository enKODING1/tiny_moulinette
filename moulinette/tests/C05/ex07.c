#include <stdio.h>

int ft_find_next_prime(int nb);

int main()
{
    for (int i = -4; i <= 10; i++)
        printf("%d\n", ft_find_next_prime(i));
    printf("%d\n", ft_find_next_prime(2147483641));
    printf("%d\n", ft_find_next_prime(2147483643));
    printf("%d\n", ft_find_next_prime(2147483645));
    printf("%d\n", ft_find_next_prime(2147483647));
    return 0;
}
