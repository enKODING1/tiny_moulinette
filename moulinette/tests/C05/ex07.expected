2
2
2
2
2
2
2
3
5
5
7
7
11
11
11
2147483647
2147483647
2147483647
2147483647
